health/warnings.go

```diff
@@ -18,17 +21,37 @@ var updateAvailableWarnable = Register(&Warnable{
 	Title:    "Update available",
 	Severity: SeverityLow,
 	Text: func(args Args) string {
-		return fmt.Sprintf("An update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		if version.IsMacAppStore() || version.IsAppleTV() {
+			return fmt.Sprintf("An update from version %s to %s is available. Open the App Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsMacSys() {
+			return fmt.Sprintf("An update from version %s to %s is available. Update now from the Tailscale settings.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsWindowsGUI() {
+			return fmt.Sprintf("An update from version %s to %s is available. Update now from the Tailscale menu.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if runtime.GOOS == "android" {
+			return fmt.Sprintf("An update from version %s to %s is available. Open the Play Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else {
+			return fmt.Sprintf("An update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		}
 	},
 })
 
 // securityUpdateAvailableWarnable is a Warnable that warns the user that an important security update is available.
 var securityUpdateAvailableWarnable = Register(&Warnable{
 	Code:     "security-update-available",
 	Title:    "Security update available",
-	Severity: SeverityHigh,
+	Severity: SeverityMedium,
 	Text: func(args Args) string {
-		return fmt.Sprintf("An urgent security update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		if version.IsMacAppStore() || version.IsAppleTV() {
+			return fmt.Sprintf("An urgent security update from version %s to %s is available. Open the App Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsMacSys() {
+			return fmt.Sprintf("An urgent security update from version %s to %s is available. Update now from the Tailscale settings.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsWindowsGUI() {
+			return fmt.Sprintf("An urgent security update from version %s to %s is available. Update now from the Tailscale menu.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if runtime.GOOS == "android" {
+			return fmt.Sprintf("An urgent security update from version %s to %s is available. Open the Play Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else {
+			return fmt.Sprintf("An urgent security update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		}
```

Comment:
"urgent"? Is that more than a "security update"? Or are all security updates urgent? In which case, it's redundant?

Reply:
dropped, as you pointed out it was redundant